gnome-initial-setup/gis-driver.c

```diff
@@ -862,6 +862,31 @@ window_realize_cb (GtkWidget *widget, gpointer user_data)
   update_screen_size (GIS_DRIVER (user_data));
 }
 
+static void
+emit_startup_sound (GApplication * app, gpointer user_data)
+{
+  GDBusProxy *proxy;
+  GError *error = NULL;
+
+  proxy = g_dbus_proxy_new_for_bus_sync (G_BUS_TYPE_SESSION,
+                                         G_DBUS_PROXY_FLAGS_NONE,
```

Comment:
Use `G_DBUS_PROXY_FLAGS_DO_NOT_LOAD_PROPERTIES | G_DBUS_PROXY_FLAGS_DO_NOT_CONNECT_SIGNALS` here to save a bit of wasted work.